.github/workflows/deployProd.yml

```diff
@@ -65,6 +65,6 @@ jobs:
         with:
           creds: ${{ secrets.AZURE_CREDENTIALS }}
       - name: Deploy frontend app
-        run: az storage blob upload-batch -s build/ -d '$web' --account-name simplereportprodapp --destination-path '/app'
+        run: az storage blob upload-batch -s build/ -d '$web' --account-name simplereport${{ matrix.deploy_env }}app --destination-path '/app'
```

Comment:
How was this working before when it was using `prod` for all envs?

Reply:
It wasn't working

Reply:
I made this change this morning. so it hasn't been live for a long time